Dockerfile

```diff
@@ -1,24 +1,43 @@
 FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
+ARG VERSION_INFO=noVersion
+
+ENV NPM_CONFIG_LOGLEVEL warn
+ENV PORT=1337
 
 # Set working dir as /app
 WORKDIR /app
 
-# Add sources to /app
-COPY . .
```

Comment:
Running `COPY . .` is IMHO an anti-pattern. When building a docker image instructions (e.g. `RUN`, `COPY`, etc.) will be cached as a 'layer' in the image. When running `COPY . .` any file changing invalidates the cache for this layer. The restructured so files that likely change more often (e.g. `frontend/` or `backend/`) are part of later layers. This means that if you're only changing the code in `frontend/` or `backend/`, repeated builds of this image will take advantage of the caching of earlier layers and save you re-building them.